### ISSUE TITLE
How to include cash in assets?

### ISSUE THREAD

**Issue by TradeProphet** [robertmartin8/PyPortfolioOpt#196]
I'm using the library to optimize a number of strategies but I would like to include cash with the strategies performance metrics so the weights would include the option to NOT be fully invested but also go to cash in times of higher risk

**Comment by robertmartin8** [comment]
Hi @TradeProphet,

Thanks for the question and apologies for the slow reply.

As discussed in #143, you can create a cash asset by creating a column with a constant price, e.g:

```python
prices["cash"] = 1   # dummy column
```

This should work with the optimiser. Let me know how it goes!

Best,
Robert

**Comment by TradeProphet** [comment]
Hello Robert,
Thank you for your answer
I had done something similar by adding a column with the values of BND as
cash, it works the same,

Thank you again,
Alon

**Comment by robertmartin8** [comment]
I'm really glad you managed to get it to work! Don't hesitate to reach out if you have another question.

Cheers,
Robert

**Comment by TradeProphet** [comment]
I do have one more, sometimes I get None value returned as allocation
weights from the package which of course are unusable,
Would you like me to send you the complete code for testing?
Thanks,
Alon

**Comment by bruppfab** [comment]
**What are you trying to do?**

I am trying to optimize a portfolio with pf.efficient_risk() and a target volatility of 8%. If, however, a vola of 8% is not possible, the optimizer should choose to go in cash, for as much percent as needed to get to the target volatility. 

**What have you tried?**

I have tried adding cash as an asset as described in issue #196. Unfortunately, when trying this, I get the following error: "LinAlgError: Singular matrix" from "numpy\linalg\linalg.py"

I have tried to do the change described in pull request #434 but that results in "nan" or inaccurate weights, this warning is being returned:

"UserWarning: Solution may be inaccurate. Try another solver, adjusting the solver settings, or solve with verbose=True for more information."

Can anyone help with this?
